### PR TITLE
Fix types

### PR DESCRIPTION
**Pull request by olliechick**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ class ShopifyToken {
   /**
    * Verify the hmac returned by Shopify.
    *
-   * @param {Object} query The query string object
+   * @param {Object} query The query object, e.g. `{hmac: "fd4c", timestamp: 1337178173}`
```
Comment (lpinca) [inline index.js:106]:
```suggestion
   * @param {Object} query The parsed query string
```

File: types/index.d.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ declare class ShopifyToken {
   /**
    * Verify the hmac returned by Shopify.
    *
-   * @param {Object} query The query string object
+   * @param {Object} query The query object, e.g. `{hmac: "fd4c", timestamp: 1337178173}`
```
Comment (lpinca) [inline types/index.d.ts:52]:
```suggestion
   * @param {Object} query The parsed query string
```

File: types/index.d.ts [hunk 3]
```diff
@@ -63,7 +63,7 @@ declare class ShopifyToken {
    *     additional data
    * @public
    */
-  getAccessToken(shop: string, code: string): Promise<object>;
+  getAccessToken(shop: string, code: string): Promise<any>;
```
Comment (lpinca) [inline types/index.d.ts:66]: Why this change?

Comment (lpinca) [reply types/index.d.ts:66]: Ah ok, I didn't read commit message.

Comment (lpinca) [reply types/index.d.ts:66]: Please wrap commit message body to 72 chars per line.

File: index.js (hunk 1 above)
Comment (olliechick) [reply index.js:106]: I think that makes it sound like it expects a string as input

Comment (lpinca) [reply index.js:106]: Type is specified. `Object`.

File: types/index.d.ts (hunk 3 above)
Comment (lpinca) [reply types/index.d.ts:66]: Can we use `Record<string, string>` instead of `any`?

File: index.js (hunk 1 above)
Comment (olliechick) [reply index.js:106]: It's not clear that is doesn't expect a `String` object though, like `String("hmac=fd4c&timestamp=1337178173")`

Comment (lpinca) [reply index.js:106]: The same applies to the ctor options object then. It does not make much sense. It type was a string it would have been `@param {String}` like other string parameters.

File: types/index.d.ts (hunk 3 above)
Comment (olliechick) [reply types/index.d.ts:66]: yup

**Comment by lpinca** [comment]
Can you please use `[ts]` instead of `[fix]` for the TypeScript definition change?

**Comment by lpinca** [comment]
Do you mind removing first commit or accept the suggestion?

File: index.js (hunk 1 above)
Comment (olliechick) [reply index.js:106]: ok

Comment (lpinca) [reply index.js]: Also there is an example in the [README](https://github.com/lpinca/shopify-token#shopifytokenverifyhmacquery) that make it very clear in my opinion.

**Comment by lpinca** [comment]
Please use `[doc]` for first commit, then I'm done with nits.

**Comment by olliechick** [comment]
> Please use `[doc]` for first commit title, then I'm done with nits.

Done, thanks for your review

**Review by lpinca — APPROVED** [review]

**Comment by lpinca** [comment]
Thank you.
